Gentoo bug: https://bugs.gentoo.org/show_bug.cgi?id=443730

From 1c6656df4c4bf342133acec0ef3bf26b1ba40f8f Mon Sep 17 00:00:00 2001
From: Michael Palimaka <kensington@gentoo.org>
Date: Thu, 6 Dec 2012 02:38:05 +1100
Subject: [PATCH] Update included .desktop file.

This removes the deprecated 'Application' category, and adds the more detailed 'Player' category.
---
 minitube.desktop |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/minitube.desktop b/minitube.desktop
index 8016dd2..63df494 100644
--- a/minitube.desktop
+++ b/minitube.desktop
@@ -16,7 +16,7 @@ Exec=minitube
 Terminal=false
 Type=Application
 Icon=minitube
-Categories=Application;Qt;AudioVideo;Video;
+Categories=Qt;AudioVideo;Video;Player;
 StartupNotify=true
 Actions=TogglePlaying;Next;Previous;
 
-- 
1.7.1

